scripts/osm2pgsql-replication

```diff
@@ -115,6 +134,84 @@ def update_replication_state(conn, table, seq, date):
 
     conn.commit()
 
+def status(conn, args):
+    """\
+    Print information about the current replication status
+    """
```

Comment:
I could swear the initial version had more documentation in the man page. Extended documentation (including a description of the json output should be added here and will then automatically appear in the man page.

Also: please finish the sentence with a full stop.

Reply:
You can see the original version here. It had the same (low level of) detail https://github.com/openstreetmap/osm2pgsql/commit/eb4c79c4625c9fc384a3e1701c9f3a4fd07194f6